MIGRATION.md

```diff
@@ -0,0 +1,1272 @@
+# Sentry Capacitor and JavaScript SDK Migration Docs
+
+These docs walk through how to migrate our Capacitor and JavaScript SDKs.
+The documentation will guide you to upgrade @Sentry/Capacitor along with the siblings,
+
+# Upgrading from Sentry Capacitor 0.x to 1.x
+
+- With this change, we will no longer support Angular 13, if you need to stay on Angular 13, you should stay on version 0.X and `@Sentry/Angular` on version 7.x.
+
+- If you use the package `@sentry/angular-ivy`, you will need to replace it by `@sentry/angular`.
+
+- enableOutOfMemoryTracking was removed and replaced by enableWatchdogTerminationTracking.
```

Comment:
The same as for the changelog, we can just point to the JS file.

And the capacitor changes looks small, those might fit in to the changelog, if not we can have a small migration md for cap.